modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -797,6 +797,51 @@ GAPI_FLUID_KERNEL(GFluidDivRC, cv::gapi::core::GDivRC, false)
     }
 };
 
+//-------------------
+//
+// Fluid kernels: mask
+//
+//-------------------
+
+template<typename DST, typename SRC>
+static void run_mask(Buffer &dst, const View &src, const View &mask)
+{
+    static_assert(std::is_same<DST, SRC>::value, "wrong types");
```

Comment:
Please, provide more descriptive comment (e.g. `Input and output types should match`)

Reply:
OK, changed